### ISSUE TITLE
Clarification of "democratic processes"

### ISSUE THREAD

**Issue by realpixelcode** [raisely/NoHarm#71]
> 5. This software must not be used by any organisation, website, product, or service that:
> ii. lobbies against, or derives a majority of income from actions that discourage or frustrate:
> - democratic processes

The rule refers exclusively to "democratic processes", basically elections. However, I think that this licence should protect democracy and freedom as such, not just individual components. This would be possible, for example, by including the principle of the "free democratic basic order" as an asset to be preserved. This principle is bindingly laid down in the German constitution (see Art. 18 GG) and was defined in a ruling of the Federal Constitutional Court as follows:

> "A liberal democratic basic order within the meaning of Article 21 II of the Basic Law is an order which, to the exclusion of any rule by force or arbitrary power, constitutes a rule of law based on the self-determination of the people according to the will of the respective majority and on freedom and equality. The fundamental principles of this order include at least: respect for the human rights concretised in the Basic Law, above all for the right of the personality to life and free development, the sovereignty of the people, the separation of powers, the accountability of the government, the lawfulness of the administration, the independence of the courts, the multi-party principle and equality of opportunity for all political parties with the right to form and exercise an opposition in accordance with the constitution."

I find this to be exactly in keeping with the nature of the licence and thus propose the following amendment to the above rule:

- democracy, democratic processes, the rule of law, the separation of powers or the free democratic basic order of a state.

**Comment by chrisjensen** [comment]
> “democracy is the worst form of government – except for all the others that have been tried.”

While we're certainly supporters of democratic government we wanted to stop short of prescribing that no-one should work towards anything other than democracy. At the same time, we wanted to protect unions, which are also a kind of democracy, from being busted, so we thought this wording would strike a balance between those two ideas.

As I mentioned in the other issue, Rule of Law is problematic as activists working in oppressive regimes that make their very existence illegal could be construed as frustrating the rule of law in that country.

**Comment by realpixelcode** [comment]
Well, with "rule of law" I didn't really mean that "the law is obeyed". In my mother tongue German, there's the great term "Rechtsstaat" or "Rechtsstaatlichkeit" which is usually translated with "rule of law" but that doesn't cover the full range of the original term. "Rechtsstaatlichkeit" is not necessarily about obeying law but about the following circumstances:

- every citizen has the constitutional right to enforce one's legal rights in independent courts with "effective legal effect"
- politics and government are accountable to the law
- human rights generally apply

It can be explained better using an example of "Unrechtsstaatlichkeit" or "absence of the rule of law", in particular the former German Democratic Republic. Politics governed the law, citizens had no option to enforce the rights they got on paper, the rights they got were worth nothing as courts weren't independent, police could arrest anyone without justification, anyone could be spied on without justification and so on and so on.

Edit: In German, "Rechtsstaatlichkeit" or "rule of law" is understood as the exact opposite of "arbitrary rule".

**Comment by ghost** [comment]
I just thought I'd help out here, but based what I'm reading, this discussion is getting a bit country-centric. Based on my knowledge on the topic, I can point out that most democracies have a sort of bicameral (i.e. meaning two chambers [note 1]) structure for writing legislation (i.e. laws), and have the President/P.M./Chancellor as the head of the executive branch that enforces the laws [note 2].

[note 1]: Depending on the country, the names of each chamber will have different names: Senate and House of Representatives for the U.S. and the houses of Lords and Commons for the U.K.
[note 2]: Some democracies are constitutional monarchies that have the "royal assent" where bills are sent to the monarch to be enacted as laws. This is what the U.K. has, and the process is similar to what happens with the President of the United States.

**Comment by realpixelcode** [comment]
Yeah, that's, in fact, also true for Germany. We have the Bundestag that passes laws and that is elected by the voters, and there's the Bundesrat that has a veto and that consists of representatives from the Federal States' governments. The Federal President ratifies (has to ratify) all laws.

**Comment by ghost** [comment]
@realpixelcode, I think we need to leave rule of law alone right now because that is what is keeping the democratic world afloat.
> The Rule of Law permeates all aspects of American life. For example, we have traffic laws that let us know who has the right of way and we have environmental laws and regulations that tell us what we are allowed to put into the ground, air and water.

[[source](https://www.americanbar.org/groups/public_education/law-day/law-day-2021/how-the-rule-of-law-has-shaped-and-continues-to-shape-america/)]
